tests/UnifiedSpecTests/command-monitoring/writeConcernError.json

```diff
@@ -0,0 +1,155 @@
+{
+  "description": "writeConcernError",
+  "schemaVersion": "1.12",
```

Comment:
The schema version is downgraded from `1.13` to `1.12`: this file uses none of the new fields of `1.13`: `observeLogMessages`, `logComponent`, `logSeverityLevel`, `expectedLogMessagesForClient`, `expectLogMessages`.

Reply:
This should be reverted to 1.13 for now. We don't modify the JSON test files when syncing them to the specs repo. Some other drivers, like libmongoc, do that, but IMO it becomes problematic as we're then unable to differentiate driver-specific changes from upstream changes.

Correcting this would be better done upstream. That would entail the overhead of a DRIVERS ticket, but you could avoid spawning language tickets by selecting "Driver Changes: Not Needed" like I did in [DRIVERS-2641](https://jira.mongodb.org/browse/DRIVERS-2641).

@alcaeus and I were actually discussing this last week based on a comment I left in https://github.com/mongodb/specifications/pull/1423#discussion_r1200688693 (another spec test using a higher schema version than required). He'd like to explore adding a CI step to the specs repo to catch this and I assume the implementation would also entail revising existing spec test files. He'll likely create a DRIVERS ticket to track that when he's back from PTO, so I think we can just wait on that (and avoid fixing this test file upstream for now).

Reply:
I cannot revert the schemaVersion without updating `MAX_SCHEMA_VERSION`. Which is even worse if the new fields are not implemented in the test runner.
https://github.com/mongodb/mongo-php-library/blob/f6551edd60a368f019bb7cd1dda4fe78a5e7597b/tests/UnifiedSpecTests/UnifiedTestRunner.php#L64-L66

Reply:
Noted. I think it's best to get this addressed upstream first, then. You can follow the process I used in [DRIVERS-2641](https://jira.mongodb.org/browse/DRIVERS-2641) to avoid extra language tickets.

Reply:
New ticket DRIVERS-2644 and PR https://github.com/mongodb/specifications/pull/1431